Android.gitignore

```diff
@@ -34,13 +34,7 @@ captures/
 
 # IntelliJ
 *.iml
-.idea/workspace.xml
-.idea/tasks.xml
-.idea/gradle.xml
-.idea/assetWizardSettings.xml
-.idea/dictionaries
-.idea/libraries
-.idea/caches
+.idea/
```

Comment:
I don't agree with this bit

>  all these files are generate we could simply ignore all.

We are tracking a separate `Global/JetBrains.gitignore` which refers to [this documentation](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839) from JetBrains that indicates there are many files under `.idea` that can be committed to version control.